docs/mllib-evaluation-metrics.md

```diff
@@ -0,0 +1,1476 @@
+---
+layout: global
+title: Evaluation Metrics - MLlib
+displayTitle: <a href="mllib-guide.html">MLlib</a> - Evaluation Metrics
+---
+
+* Table of contents
+{:toc}
+
+
+## Algorithm Metrics
```

Comment:
Maybe we can remove the section title. I was wondering how `Algorithm Metrics` differs from `Evaluation Metrics` when I first saw it.